funnel/forms/profile.py

```diff
@@ -24,8 +24,9 @@ class ProfileForm(OrganizationForm):
     A `profile` keyword argument is necessary for the ImgeeField.
     """
 
-    __expects__ = ('profile',)
+    __expects__ = ('profile', 'user')
     profile: Profile
+    user: User
```

Comment:
Get user from profile.user. It's not available when the form is used for an organization.